src/app/(BottomTabNavigation)/Cases/CaseSummaryScreen/index.tsx

```diff
@@ -0,0 +1,41 @@
+import { useLocalSearchParams } from 'expo-router';
+import React from 'react';
+import { View, Text, Image, ScrollView } from 'react-native';
+
+import styles from './styles';
+import ExternalSiteLink from '../../../../Components/ExternalSiteLink/ExternalSiteLink';
+import { CaseSummaryProps } from '../../../../types/types';
+import { formatDate } from '../utils';
+
+export default function CaseSummaryScreen() {
+  const caseData = useLocalSearchParams() as unknown as CaseSummaryProps;
+
+  return (
+    <View style={styles.container}>
+      <ScrollView
+        style={styles.scrollContainer}
+        showsVerticalScrollIndicator={false}
+      >
+        <Image
+          style={styles.imageContainer}
```

Comment:
Let's use the image from supabase storage! The images are stored by case id and we will need to use a combination of `useEffect` and `useState` here. I will explain this to everyone on Sunday so maybe best to hold off till then!

Reply:
I will give it a try but I'll lyk if I can't figure it out.